README.md

```diff
@@ -21,11 +21,24 @@ This work aims to port PyTorch to HammerBlade.
  - Build pytorch. This step can take up to 15 minutes
     `python setup.py install`
 
+### How to build PyTorch with Emulation Layer
+    `git clone -b hb-device git@github.com:cornell-brg/hb-pytorch.git`
+ - Create python virtual environment
+    `python3.6 -m venv ./venv_pytorch`
+    `python3.6 -m venv ./venv_pytorch`
+ - Install dependencies
+    `pip install numpy pyyaml mkl mkl-include setuptools cmake cffi typing sklearn tqdm pytest`
+ - Init pytorch third party dependencies
+    `git submodule update --init --recursive`
+ - Setup building environment variables.
+    `cd hb-pytorch && source setup_emul_build_env.sh`
+ - Build pytorch. This step can take up to 15 minutes
+    `python setup.py install`
```

Comment:
A minor thing... `python setup.py develop` is faster for incremental builds. `develop` only creates symlinks for python modules whereas `install` copies them.

Reply:
good point! I'll switch to use `develop`